webhook/certificates/certificates.go

```diff
@@ -68,7 +71,18 @@ func (r *reconciler) reconcileCertificate(ctx context.Context) error {
 		logger.Infof("Certificate secret %q is missing key %q", r.secretName, certresources.CACert)
 	} else {
 		// It has all of the keys, it's good.
-		return nil
+		cert, err := tls.X509KeyPair(secret.Data[certresources.ServerCert], secret.Data[certresources.ServerKey])
+		if err != nil {
+			logger.Infof("error creating pem from certificate and key")
+		}
```

Comment:
everything below this "if" should be wrapped in an else block to avoid errors due to the failed condition.